dashboard/src/components/entitypage/elements/MetricsDropdown.js

```diff
@@ -140,10 +140,16 @@ const MetricsDropdown = ({ type, name, variant, timeRange, aggregates }) => {
       window.location.href = createPromUrl();
     }
   };
+  let totalChartHeight = 0;
+  const queryObj = queryFormats[type];
+  if (queryObj) {
```

Comment:
does it make sense to add a test case for this?

Reply:
Good idea, I could probably break this out to a function and point a test at it.

Reply:
done.